files/SetupSafeHavenIAM.json

```diff
@@ -0,0 +1,1573 @@
+
+
+
+
+
+
+<!DOCTYPE html>
```

Comment:
This does not appear to be a JSON file. Please either change the contents or the file extension.